cbmc/proofs/mqtt_state.c

```diff
@@ -182,7 +182,7 @@ bool valid_IotMqttOperationList( const IotListDouble_t *pOp,
   IotListDouble_t *pLink;
   IotContainers_ForEach( pOp, pLink ) {
     IotMqttOperation_t
-      *pElt = IotLink_Container( _mqttSubscription_t, pLink, link );
+      *pElt = IotLink_Container( struct _mqttOperation, pLink, link );
```

Comment:
You also need to update line 460.

Reply:
Why?  Line 460 is iterating over a subscription list and the type is correct.  This mistake was made iterating over an operation list.

Reply:
Oh true! Approved!

Reply:
Spacing of these 3 lines seems inconsistent.

Reply:
I'm not sure how to break the long line, I pushed a revision that doesn't break the line.